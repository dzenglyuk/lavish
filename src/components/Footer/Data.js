export const infoLinks = [
    {
        title: "About Us",
        links: [
            {
                name: "How it works",
                value: "/sign-up",
            },
            {
                name: "Testimonials",
                value: "/",
            },
            {
                name: "Careers",
                value: "/",
            },
            {
                name: "Investors",
                value: "/",
            },
            {
                name: "Terms of Service",
                value: "/",
            },
        ],
    },
    {
        title: "Contact Us",
        links: [
            {
                name: "Contact",
                value: "/",
            },
            {
                name: "Support",
                value: "/",
            },
            {
                name: "Destinations",
                value: "/",
            },
            {
                name: "Sponsorships",
                value: "/",
            },
        ],
    },
];

export const socialLinks = [
    {
        title: "Videos",
        links: [
            {
                name: "Submit Video",
                value: "/",
            },
            {
                name: "Ambassadors",
                value: "/",
            },
            {
                name: "Agency",
                value: "/",
            },
            {
                name: "Influencer",
                value: "/",
            },
        ],
    },
    {
        title: "Social Media",
        links: [
            {
                name: "Instagram",
                value: "/",
            },
            {
                name: "Facebook",
                value: "/",
            },
            {
                name: "Youtube",
                value: "/",
            },
            {
                name: "Twitter",
                value: "/",
            },
        ],
    },
];

export const subscription = {
    heading: "Join our exclusive membership to receive the latest news and trends",
    text: "You can unsubscribe at any time."
};