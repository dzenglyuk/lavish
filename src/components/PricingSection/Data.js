const starterAccount = {
    info: {
        name: "Starter",
        price: "$8.99",
        period: "per month"
    },
    features: ["100 Transactions", "2% Cash Back", "$10,000 Limit"]
};

const goldAccount = {
    info: {
        name: "Gold",
        price: "$29.99",
        period: "per month"
    },
    features: ["1000 Transactions", "3.5% Cash Back", "$100,000 Limit"]
};

const diamondAccount = {
    info: {
        name: "Diamond",
        price: "$99.99",
        period: "per month"
    },
    features: ["Unlimited Transactions", "5% Cash Back", "Unlimited Spending"]
};

export {
    starterAccount,
    goldAccount,
    diamondAccount
};