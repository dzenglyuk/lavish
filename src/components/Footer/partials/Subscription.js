import React from "react";
import Button from "../../Button/Main";
import { subscription } from "../Data";

const Subscription = ({ heading, text }) => {
    return (
        <section className="footer-subscription">
            <p className="footer-subscription-heading">{heading}</p>
            <p className="footer-subscription-text">{text}</p>
            <div className="input-areas">
                <form>
                    <input
                        className="footer-input"
                        name="email"
                        type="email"
                        placeholder="Your Email"
                    />
                    <Button btnStyle="btn--outline">Subscribe</Button>
                </form>
            </div>
        </section>
    );
};

Subscription.defaultProps = {
    heading: subscription.heading,
    text: subscription.text,
};

export default Subscription;
