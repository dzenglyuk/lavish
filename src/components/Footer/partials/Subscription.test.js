import React from "react";
import { render } from "@testing-library/react";
import renderer from "react-test-renderer";

import { subscription } from "../Data";
import Subscription from "./Subscription";

test("Renders correctly", () => {
    const { getByText } = render(<Subscription />);
    getByText(subscription.heading);
    getByText(subscription.text);
    getByText("Subscribe");
});

test("Matches the snapshot", () => {
    const tree = renderer.create(<Subscription />).toJSON();
    expect(tree).toMatchSnapshot();
});
