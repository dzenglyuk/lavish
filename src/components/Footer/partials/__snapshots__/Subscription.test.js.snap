// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `
<section
  className="footer-subscription"
>
  <p
    className="footer-subscription-heading"
  >
    Join our exclusive membership to receive the latest news and trends
  </p>
  <p
    className="footer-subscription-text"
  >
    You can unsubscribe at any time.
  </p>
  <div
    className="input-areas"
  >
    <form>
      <input
        className="footer-input"
        name="email"
        placeholder="Your Email"
        type="email"
      />
      <button
        className="btn 
			btn--outline
			btn--medium
			"
      >
        Subscribe
      </button>
    </form>
  </div>
</section>
`;
